Userland/Libraries/LibC/arch/aarch64/fenv.cpp

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2021, Mițca Dumitru <dumitru0mitca@gmail.com>
```

Comment:
The aarch64 and riscv files seems different enough that you can put either your own copyright line, or, if you don't feel comfortable doing that (but you should feel comfortable :) ), the generic 'the SerenityOS Developers` line.

the year for these new files should just be 2024.